classes/options-general.php

```diff
@@ -216,36 +216,36 @@ function print_page() {
 
 											if ( ! url_has_ftps_for_item( $simplemap_ps ) ) : ?>
 
-												<h4><?php printf( __( '<strong>SimpleMap Premium Support Benefits - $30/year/domain</strong>', 'simplemap' ), esc_attr( site_url() ) ); ?></h4>
+												<h4><?php printf( __( '<strong>SimpleMap Premium Support Benefits - $30/year/domain</strong>', 'SimpleMap' ), esc_attr( site_url() ) ); ?></h4>
 												<p>
-													<?php //printf( __( 'SimpleMap now offers a premium support package for the low cost of %s per year per domain.', 'simplemap' ), '$30.00 USD' ); ?>
+													<?php //printf( __( 'SimpleMap now offers a premium support package for the low cost of %s per year per domain.', 'SimpleMap' ), '$30.00 USD' ); ?>
 												</p>
 												<p>
-													<?php _e( '<em>By signing up for SimpleMap premium support, you help to ensure future enhancements to this excellent project as well as the following benefits:</em>', 'simplemap' ); ?>
+													<?php _e( '<em>By signing up for SimpleMap premium support, you help to ensure future enhancements to this excellent project as well as the following benefits:</em>', 'SimpleMap' ); ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 236 characters

<!-- Comment by [SideCI](https://sideci.com) -->